RDF.js

```diff
@@ -754,19 +862,22 @@ function importItem(newItem, node) {
 		if(creatorType == "author") {
 			creators = getFirstResults(node, [n.bib+"authors", n.dc+"creator", n.dc1_0+"creator",
 				n.dcterms+"creator", n.eprints+"creators_name",
-				n.dc+"contributor", n.dc1_0+"contributor", n.dcterms+"contributor"]);
+				n.dc+"contributor", n.dc1_0+"contributor", n.dcterms+"contributor",
+				n.so+"author", n.so+"creator"]);
```

Comment:
we should think about where we but so in the hierarchy of preference. I'd have said we prefer it to dc, say?

Reply:
Before `dc+"contributor"` or before `dc+"creator"`?

Reply:
I'm not sure, but my inclination would be to say the `so` comes right after `bib` and `z` in expected quality, but that could be totally off base, what do you think?

Reply:
Okay